DataSet.rst

```diff
@@ -503,27 +507,29 @@ The pattern for ``TimeStampedName`` attribute generated by production PacBio too
 <metatype>-<yymmdd_HHmmssttt>
 
 Where ``MetaType`` has been transformed into a lowercase, underscore separated
-string and the time string format directives map to the following entities\:
-year, month, day, hour, minute, second, millisecond.
+string and the time string format directives map to the following entities\: year, month, day, hour,
+minute, second, millisecond. For simplicity, indices are all given an "index" metatype prefix
```

Comment:
All the *Index* metatypes do not appear to have their own TimeStampedName prefixes,  but just use "index-" instead.